test/common/cpu-prof.js

```diff
@@ -34,23 +34,12 @@ function verifyFrames(output, file, suffix) {
   assert.notDeepStrictEqual(frames, []);
 }
 
-let FIB = 30;
-// This is based on emperial values - in the CI, on Windows the program
-// tend to finish too fast then we won't be able to see the profiled script
-// in the samples, so we need to bump the values a bit. On slower platforms
-// like the Pis it could take more time to complete, we need to use a
-// smaller value so the test would not time out.
-if (common.isWindows) {
-  FIB = 40;
```

Comment:
I'd prefer it to stay here so that it's possible to experiment with

```
FIB=ANOTHER_VALUE out/Release/node --cpu-prof test/fixtures/workload/fibonacci.js
```

Reply:
If I'm not mistaken, that works with the current change. It's just not documented in `common/README.md` (but it's not documented there right now either). See `fibonacci.js` where `process.env.FIB` takes priority over the default value `FIB`.

Reply:
Oh, yes, this still works.